content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,158 @@
+
+To avoid this, REANA 0.9.0 introduces a limit on the number of scheduling retries of each workflow. This limit is set to 200 retries by default, but it can be customized with the `reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT` Helm value.
+
+### Configure custom TLS certificates
+
+REANA 0.9.0 also improves the handling of TLS certificates.
+Without additional configuration needed, a self-signed certificate lasting 90 days is automatically generated each time REANA is deployed or upgraded.
+This is enough for development instances, but a valid certificate issued by a trusted Certificate Authority should be used for production deployments.
+
+For this reason, it is possible to disable the generation of the default self-signed certificate by setting the `ingress.tls.self_signed_cert` Helm value to false.
+You can then provide a custom TLS certificate by creating an appropriate Kubernetes TLS secret and by setting the `ingress.tls.secret_name` Helm value to the name of this newly-created secret.
+
+See the documentation page about [configuring TLS certificates](https://docs.reana.io/administration/configuration/configuring-tls-certificates) to learn more about this.
+
+### How to upgrade existing REANA 0.8 clusters
+
+If you are a REANA cluster administrator and you would like to upgrade from
+REANA 0.8.0 to REANA 0.9.0, you can optionally define the new [Helm
+values](https://github.com/reanahub/reana/blob/0.9.0/helm/reana/README.md)
+listed above in your Helm values file:
+
+```console
+$ vim myvalues.yaml
+```
+
+You can then use the [Helm diff plugin](https://github.com/databus23/helm-diff)
+to inspect the forthcoming change and then perform the upgrade using the
+standard Helm commands:
+
+```console
+$ helm repo update
+$ helm diff upgrade reana reanahub/reana --version 0.9.0 --values myvalues.yaml
+$ helm upgrade reana reanahub/reana --version 0.9.0 --values myvalues.yaml
+```
+
+This release introduces several database schema changes to support workflow
+retention rules, launching workflows from external sources and introduces some small
+improvements and fixes. We thus need to perform database schema upgrade by
+following the dedicated [upgrade documentation](http://docs.reana.io/administration/deployment/upgrading-db/):
+
+```console
+$ kubectl exec -i -t deployment/reana-server -c rest-api -- reana-db alembic upgrade
+INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+INFO  [alembic.runtime.migration] Will assume transactional DDL.
+INFO  [alembic.runtime.migration] Running upgrade c912d4f1e1cc -> ad93dae04483, Interactive sessions.
+INFO  [alembic.runtime.migration] Running upgrade ad93dae04483 -> 4801b98f6408, Job started and finished times.
+INFO  [alembic.runtime.migration] Running upgrade 4801b98f6408 -> f84e17bd6b18, Workflow complexity.
+INFO  [alembic.runtime.migration] Running upgrade f84e17bd6b18 -> 6568d7cb6710, storing full workflow workspace.
```

Comment:
```suggestion
```

Reply:
Not sure what is the difference here?

Reply:
I thought of removing those lines, as people upgrading from 0.8.0 should have already applied those revisions